content/src/ui/Component.CreateSessionModal.json

```diff
@@ -3,6 +3,21 @@
     "selectContent": {
       "title": "Pick an exercise"
     },
+    "selectType": {
+      "title": "Create session with...",
+      "public": {
+        "title": "Anyone",
+        "icon": "https://res.cloudinary.com/twentyninek/image/upload/v1664528333/temp/pure-simple-love_rd2wvb.png"
+      },
+      "private": {
+        "title": "My friends",
+        "icon": "https://res.cloudinary.com/twentyninek/image/upload/v1664528333/temp/pure-simple-love_rd2wvb.png"
+      },
+      "async": {
```

Comment:
I don't think this type will ever happen. It will be more async content than async sessions.